test/functional-tests/stepDefinitions/caseDetailsSteps.js

```diff
@@ -0,0 +1,27 @@
+let CaseDetailsPage = require('../pageObjects/caseDetailsPage.js');
+
+let chai = require("chai").use(require("chai-as-promised"));
+let expect = chai.expect;
+
+var { defineSupportCode } = require("cucumber");
+
+defineSupportCode(function ({ Given, When, Then, Before, After }) {
+
+  Then(/^the latest History event should be '(.*)'$/, async function (expectedEvent) {
+      let actualEvent = await new CaseDetailsPage().getLatestHistoryEvent()
```

Comment:
Do you think instead of creating an object of CaseDetailsPage every time, can we create the object once and use in other steps also?

Reply:
done